src/pages/Environments.js

```diff
@@ -420,36 +435,40 @@ const Environments = () => {
             field: 'project',
             headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
-              const { status: diskStatus, googleProject } = filteredDisks[rowIndex]
+              const { labels: { saturnWorkspaceNamespace } } = filteredDisks[rowIndex]
+              return saturnWorkspaceNamespace
+            }
+          },
+          {
+            field: 'workspace',
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'workspace', onSort: setDiskSort }, ['Workspace']),
+            cellRenderer: ({ rowIndex }) => {
+              const { status: diskStatus, googleProject, labels: { saturnWorkspaceNamespace, saturnWorkspaceName } } = filteredDisks[rowIndex]
               const appType = getDiskAppType(filteredDisks[rowIndex])
               const multipleDisksOfType = _.remove(disk => getDiskAppType(disk) !== appType || disk.status === 'Deleting',
                 disksByProject[googleProject]).length > 1
-              const forAppText = !!appType ? ` for ${_.capitalize(appType)}` : ''
-              return h(Fragment, [
-                googleProject,
+              return h(Fragment,
+                [h(Link, { onClick: () => Nav.goToPath('workspace-notebooks', { namespace: saturnWorkspaceNamespace, name: saturnWorkspaceName }) },
```

Comment:
Same comments as above about going to the notebooks tab vs. the Workspace dashboard, and the desire for an a-tag instead of a "button".